src/js/sonic-reveal.js

```diff
@@ -1,42 +1,42 @@
   ],
   speechiness: [
-    `<b>Speechiness</b> is a measure of the amount of spoken-word (as opposed to singing) in the track. “Believe” gets a low score here, though not quite 0 (perhaps because of the "I don't need you anymore" section of the bridge, which has a somewhat speechy feel).`
+    `<b>Speechiness</b> is a measure of the amount of spoken-word (as opposed to singing) in the track. “Believe” gets a low score here, though not quite zero (perhaps because of the “I don’t need you anymore” section of the bridge, which has a somewhat speechy feel).`
   ],
   instrumentalness: [
-    `<b>Instrumentalness</b> is the estimated likelihood that this is an instrumental track. “Believe”, with its prominent vocals throughout, gets a 0 on this.`
+    `<b>Instrumentalness</b> is the estimated likelihood that this is an instrumental track. “Believe,” with its prominent vocals throughout, gets a zero on this.`
   ],
   valence: [
-    `Finally, <b>valence</b> is one of the most high-level, abstract features used. It represents the "positiveness" of the track. That is, is listening to it likely to make you happy? Or does it have more of a sad, dark vibe?`,
+    `Finally, <b>valence</b> is one of the most high-level, abstract features used. It represents the “positiveness” of the track. That is, is listening to it likely to make you happy? Or does it have more of a sad, dark vibe?`,
     // Major except the bridge?
-    `“Believe” gets a slightly below average 48% valence score. We might agree with that based on the lyrics ("I really don't think it's strong enough"), but the algorithm only has access to the raw audio of the song, not the liner notes. Its assessment of valence comes from the song's sound.`,
+    `“Believe” gets a slightly below average 48 percent valence score. We might agree with that based on the lyrics (“I really don't think it’s strong”), but the algorithm only has access to the raw audio of the song, not the liner notes. Its assessment of valence comes from the song‘s sound.`,
    `I would tend to agree that “Believe” doesn't sound like sunshine and rainbows, but it's hard to say exactly why. It's commonly observed that songs in a 
       <a href="https://www.youtube.com/watch?v=-9YopDo5_xU">major key</a>
       tend to sound more upbeat and positive than those written in a 
       <a href="https://www.youtube.com/watch?v=jcA--c2U4Wg">minor key</a>,
-      but “Believe” is in the key of G major, so the answer isn't that simple.`,
-      `Maybe there's just something unsettling about the alien, artificial sound of those synth pads? Or maybe it's something in the harmonies. The arpeggiated chord played after the first line of the chorus sounds a bit wonky: it's a major seventh, inverted such that the first two notes are separated by a semitone - an extremely dissonant interval. Or maybe it's just Cher's creepy robot voice.`
```

Comment:
FWIW, I kind of liked the puerility of the last sentence, as a way to deflate the mildly pretentious tone of the rest of the paragraph. But I'm not married to it.